tequilapi/endpoints/service.go

```diff
@@ -70,7 +72,27 @@ func NewServiceEndpoint(serviceManager ServiceManager, optionsParser map[string]
 //     schema:
 //       "$ref": "#/definitions/APIError"
 func (se *ServiceEndpoint) ServiceList(c *gin.Context) {
-	instances := se.serviceManager.List()
+	includeAll := false
+	includeAllStr := c.Request.URL.Query().Get("include_all")
+	if len(includeAllStr) > 0 {
+		var err error
+		includeAll, err = strconv.ParseBool(includeAllStr)
+		if err != nil {
+			c.Error(apierror.BadRequestField(fmt.Sprintf("Failed to parse request: %s", err.Error()), "include_all", contract.ErrCodeServiceList))
+			return
+		}
+	}
+
+	instances := make([]*service.Instance, 0)
+	if includeAll {
```

Comment:
Could these be a method on the `serviceManager` itself? Would be nice to not have this `if / else` logic in the handler and the manager itself could decide if it should return all or some portion based on given params.

Reply:
good point, I thought about it too